crates/app/tests/mod.rs

```diff
@@ -16,6 +19,7 @@ use reqwest::StatusCode;
 async fn app() {
     let lis = TcpListener::bind((Ipv4Addr::UNSPECIFIED, 0)).unwrap();
     let addr = format!("http://{}", lis.local_addr().unwrap());
+    let test_data = b"testing";
```

Comment:
just use `b"test"` directly

Reply:
Why not have a variable, so it can be compared against later on?